### ISSUE TITLE
Double scrollbar on the log view 

### ISSUE THREAD

**Issue by tchiotludo** [kestra-io/kestra#5214]
![image](https://github.com/user-attachments/assets/6fcbdda9-1b1c-42c9-a7b9-928aba30c4f9)
![image](https://github.com/user-attachments/assets/2bac1fc1-be5b-458e-9701-b650ac469058)

The second one seems useless

**Comment by MilosPaunovic** [comment]
One scrollbar is for the list of tasks, and the other one is for the list of logs of a certain task, because the height is limited. I can remove the height limit for task logs, but, if a task has plenty of them, it's going to be a lot of scrolling necessary.

@tchiotludo Do we keep it like this or do we remove the height limit?

**Comment by tchiotludo** [comment]
the height limit on log is normal, I'm mostly thinking to remove the others one that I didn't understand

**Comment by MilosPaunovic** [comment]
The one to the right is for the scrolling through tasks, and the shorter, wider one is for scrolling through the logs of an expanded task.

![image](https://github.com/user-attachments/assets/7f448951-ad7a-4142-8c09-677501ab43bd)

Where does this leave us, are we going to keep it like this or do we need a change of some sort?

**Comment by Ben8t** [comment]
Agree with @MilosPaunovic  - not sure to understand the issue @tchiotludo. One scrollbar for tasks, one for logs (only appear on long logs, which seems nice).

**Comment by tchiotludo** [comment]
I asked to remove the custom scrollbar and use the native one : 
![image](https://github.com/user-attachments/assets/33e625ca-af79-4b7f-8241-fbb8ab7acfcb)
on your screen we have 3 levels of scrollbars: 
![image](https://github.com/user-attachments/assets/5e40575a-bb72-4245-bb80-951ab0a83680)
Just use the native scrollbar to scroll between task, not a custom one: 
![image](https://github.com/user-attachments/assets/05c94610-a473-45de-9fc6-da4112f0c8fb)

**Comment by Ben8t** [comment]
Indeed, @MilosPaunovic it's better highligthed when zooming the browser view
